streams/src/main/java/org/apache/kafka/streams/processor/BatchingStateRestoreCallback.java

```diff
@@ -38,4 +38,8 @@
      */
     void restoreAll(Collection<KeyValue<byte[], byte[]>> records);
 
+    @Override
+    default void restore(byte[] key, byte[] value) {
```

Comment:
Minor improvement such that any impls of the batching interface do not have to restore the per-entry `restore` func.